classes/02class/exercises/c02-network06/mfreitassm/README.md

```diff
@@ -0,0 +1,102 @@
+# c02-network06
+
+## Commands Execution Output
+
+- Commands for creating the NAT gateway:
+```
+Create a network interface
+//subnet public-b
+aws ec2 create-network-interface --subnet-id subnet-0dd087be76dd34768 --description "devopsacademy network interface" --groups sg-021fa8202867b5ce0 
+
+{
+    "NetworkInterface": {
+        "AvailabilityZone": "ap-southeast-2b",
+        "Description": "devopsacademy network interface",
+        "Groups": [
+            {
+                "GroupName": "default",
+                "GroupId": "sg-021fa8202867b5ce0"
+            }
+        ],
+        "InterfaceType": "interface",
+        "Ipv6Addresses": [],
+        "MacAddress": "02:07:d3:21:35:2e",
+        "NetworkInterfaceId": "eni-06bc2518d39cdf022",
+        "OwnerId": "490308117505",
+        "PrivateIpAddress": "10.0.10.174",
+        "PrivateIpAddresses": [
+            {
+                "Primary": true,
+                "PrivateIpAddress": "10.0.10.174"
+            }
+        ],
+        "RequesterId": "AIDAXEKFEEQA4TPKGONJ2",
+        "RequesterManaged": false,
+        "SourceDestCheck": true,
+        "Status": "pending",
+        "SubnetId": "subnet-0dd087be76dd34768",
+        "TagSet": [],
+        "VpcId": "vpc-04426efcf5b2228a4"
+    }
+}
+
+Associate Network Interface with te elastic IP
+aws ec2 associate-address --allocation-id eipalloc-0ce4bab5ca01b643f --network-interface-id eni-06bc2518d39cdf022 --private-ip-address 10.0.10.174
```

Comment:
where does `eipalloc-0ce4bab5ca01b643f` comes from?

Reply:
I forgot to put on the file.